packages/flame/lib/src/effects/color_effect.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+
+import '../../components.dart';
+import 'component_effect.dart';
+import 'controllers/effect_controller.dart';
+
+/// Change the color of a component over time.
+///
+/// This effect applies incremental changes to the component's color, and
+/// requires that any other effect or update logic applied to the same component
+/// also used incremental updates.
```

Comment:
I think this section can be removed, or replaced with a note about that the effect doesn't mix (since it isn't using incremental updates).